extensions/barrage/src/main/java/io/deephaven/extensions/barrage/util/BarrageUtil.java

```diff
@@ -657,10 +654,10 @@ private static ArrowType arrowTypeFor(Class<?> type) {
                     return Types.MinorType.LIST.getType();
                 }
                 if (type == LocalDate.class) {
-                    return LOCAL_DATE_TYPE;
+                    return Types.MinorType.DATEDAY.getType();
                 }
                 if (type == LocalTime.class) {
-                    return LOCAL_TIME_TYPE;
+                    return Types.MinorType.TIMENANO.getType();
```

Comment:
`TimeNanoVector` maps to `Long`, not `LocalTime`. Are we sure it's local, and not implicitly UTC?

Reply:
```
enum TimeUnit: short { SECOND, MILLISECOND, MICROSECOND, NANOSECOND }

/// Time is either a 32-bit or 64-bit signed integer type representing an
/// elapsed time since midnight, stored in either of four units: seconds,
/// milliseconds, microseconds or nanoseconds.
///
/// The integer `bitWidth` depends on the `unit` and must be one of the following:
/// * SECOND and MILLISECOND: 32 bits
/// * MICROSECOND and NANOSECOND: 64 bits
///
/// The allowed values are between 0 (inclusive) and 86400 (=24*60*60) seconds
/// (exclusive), adjusted for the time unit (for example, up to 86400000
/// exclusive for the MILLISECOND unit).
/// This definition doesn't allow for leap seconds. Time values from
/// measurements with leap seconds will need to be corrected when ingesting
/// into Arrow (for example by replacing the value 86400 with 86399).
table Time {
  unit: TimeUnit = MILLISECOND;
  bitWidth: int = 32;
}
```

It looks correct.
```java
    TIMENANO(new Time(TimeUnit.NANOSECOND, 64)) {
```